homeassistant/components/switch/rest.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+"""
+homeassistant.components.switch.rest
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Allows to configure a REST switch.
+
+Configuration:
+
+switch:
+  platform: rest
+  name: "Bedroom Switch"
+  resource: "http://IP_ADDRESS/ENDPOINT"
+  body_on: "ON"
+  body_off: "OFF"
+
+Variables:
+
+resource
+*Required*
+
+name
+*Optional
+The name of the switch. Default is 'REST Switch'.
+
+body_on
+*Optional
+The body that represents enabled state. Default is "ON".
+
+payload_off
```

Comment:
Yes it should be. I first had it named payload like the mqtt switch, but body made more sense in a REST context.
Thanks for pointing it out.